cli/engine/ansible/AnsibleVarsGenerator.py

```diff
@@ -69,7 +69,7 @@ def generate(self):
             # are not compatible with the new ones, defaults are used for template processing
             roles_with_defaults = [
                 'repository', 'image_registry', 'node_exporter',
-                'postgresql', 'kafka_exporter', 'jmx_exporter'
+                'postgresql', 'kafka_exporter', 'jmx_exporter', 'postgresql_exporter'
```

Comment:
Are you sure new defaults are needed for postgresql exporter? If that would be the case I would expect different configurations in the defaults

Reply:
Now we don't use specification in upgrade mode but it looks like we need update systemd service definition via template also for upgrade mode. The template uses specification. So I would keep this change to always provide defaults but we need additional task to replace the service definition during upgrade. If one need custom config, they may use `-f` option to provide it.